src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerExtensions.cs

```diff
@@ -16,10 +16,15 @@ namespace Microsoft.Extensions.Logging
     [UnsupportedOSPlatform("browser")]
     public static class ConsoleLoggerExtensions
     {
+        internal const string TrimmingRequiresUnreferencedCodeMessage = "The type of TOptions cannot be statically analyzed so its members may be trimmed.";
```

Comment:
(nit) `private`

Reply:
It actually does have to be internal as I use it in line 171 which is in a different class.

Reply:
Oh, weird. Good call.